bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,34 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        if not allow_trailing_blanklines:
+            line_nr_start = next(
+                    (i
+                     for i, line in reversed(list(enumerate(file, start=0)))
+                     if line.strip() != ''), -1) + 1
+            if line_nr_start < len(file):
+                additional_info_text = (
+                    'Your source code contains trailing blanklines at EOF.'
+                    'Those usually have no meaning. Please consider'
+                    'removing them.')
+                diff = Diff(file)
+                diff.delete_lines(line_nr_start + 1, len(file))
+                yield Result.from_values(
+                    self,
+                    'Trailing blanklines found.',
+                    diffs={filename: diff},
+                    file=filename,
+                    line=line_nr_start + 1,
+                    column=1,
+                    end_line=len(file),
+                    end_column=len(file[-1]),
+                    additional_info=additional_info_text)
+
+        else :
```

Comment:
Fix this whitespace issue @khanchi97. Run `coala` over your local repository before pushing it to remote 😉